tutorials/s4sdk-odata-service-cloud-foundry/s4sdk-odata-service-cloud-foundry.md

```diff
@@ -181,15 +188,15 @@ public class BusinessPartnerServlet extends HttpServlet {
 }
 ```
 
-The code is fairly simple. In the servlet GET method, we initialize an instance of the `BusinessPartnerService` to prepare the query to S/4HANA with the help of the SDK's Virtual Data Model. We call the method `getAllBusinessPartners`, which represents the operation of the OData service that we want to call. WIth the fluent query helper returned by this method, we can gradually build up the query:
+In the servlet GET method, we initialize an instance of the `BusinessPartnerService` to prepare the query to S/4HANA with the help of the SDK's Virtual Data Model. We call the method `getAllBusinessPartners`, which represents the operation of the OData service that we want to call. With the fluent query helper returned by this method, we can gradually build up the query:
 
 - we select the fields of the `BusinessPartner` that we want to retrieve (if we leave out this part, all fields will be returned),
 - filter for business partners of category Person (code `"1"`), and
 - order by the last name of the business partner.
+- limit the amount of results to a maximum of 200
 
 Finally, having prepared the query, we call the `execute` method. This method does a lot of the heavy lifting necessary to connect to an S/4HANA system and relieves us as developers from dealing with complex aspects such as:
 
-- the configuration which system to connect to (in a multi-tenant environment) – by transparently accessing the destination service of SAP Cloud Platform,
```

Comment:
Why has this been removed? The fact that the SDK retrieves any configuration itself, transparently, is very helpful, as is the fact that it considers multi-tenancy. The latter could be split into a separate bullet point.

Reply:
It is explained at a different place, because _which system_ to connect to is passed explicitly by the destination we pass in the `execute` call.